guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+
+2. **Prominent Metadata Display**: Use metadata fields like mint name, icon, URL, and description, to give users insight into each mint or federations identity. Display this information prominently, so users can easily verify the authenticity and relevance of each mint or federation.
+
+3. **Quick Mint Access**: Provide easy access to mint or federation details directly from the mint or federation list. This allows users to interact with mints or federations more fluidly without needing to navigate through multiple screens.
+
+#### Pending Tokens
+{% include image-gallery.html pages = page.images_pending %}
+
+It's useful to display generated tokens in the transaction history along with their status—whether pending or redeemed. This helps users quickly assess their transaction status. Providing the ability to expand on a token to view details and take actions like resending or redeeming is also important. This feature becomes particularly valuable if a token was lost or never claimed.
+
+## Cashu Best Practices
+
+### User Interface and Experience
+
+#### Default Mint
+{% include image-gallery.html pages = page.images_mint-default %}
+
+When designing for Cashu, consider allowing users to set a default mint. Since Cashu mints are independently operated, users may prefer to use a trusted mint for their transactions. By allowing them to select a default mint, you simplify their experience and reduce the risk of exposure to a potentially malicious mint. Additionally, consider offering auto-swap preferences.
+
+#### Edit Mint URL & Refresh Mint Settings
+
+{% include image-gallery.html pages = page.images_mint-url-edit %}
+
+It's useful to provide users with the ability to manually update or refresh mint settings in their wallets. While best practices suggest that wallets should auto-refresh and update mint settings periodically, there may be cases where this isn't supported, or a mint undergoes significant updates, such as a URL change. By allowing users to edit the mint URL or refresh settings manually, you ensure that users always have the latest mint configurations.
+
+### Security and Privacy
+
+#### Backup and Restore for Cashu
+The Cashu backup and restoration process is designed to ensure users can securely recover their wallets and maintain access to their bitcoin-backed ecash tokens in the event they switch devices or experience data loss. **Users should only use their recovery seed phrase once.** Repeated use of the seed phrase for restoration can lead to synchronization issues and potential errors. This is due to the fact that each time you restore, you might be dealing with an outdated state of the wallet, which can cause discrepancies in token balances and transactions.
```

Comment:
My original understanding was incorrect and was clarified by @thesimplekid. Synchronization issues may only occur if a user is using the same seed phrase across multiple devices. I'll update this with a note that says:

"Running two apps simultaneously with the same seed is not recommended, as it may cause synchronization issues."